packages/integrations/netlify/src/netlify-functions.ts

```diff
@@ -1,12 +1,9 @@
-import { polyfill } from '@astrojs/webapi';
+import '@astrojs/webapi/polyfill-ssr.js';
```

Comment:
IIUC netlify [handles dependencies this way](https://docs.netlify.com/functions/build/?fn-language=ts#manage-dependencies), if the build build script uses pnpm, and since `@astrojs/webapi` isn't a direct dep in a user's project by default, there's a chance it might break? 🤔 

Maybe we should re-export the polyfill from `@astrojs/netlify/polyfill` like the other (removed) change.

Reply:
That's what I tried first, but then I couldn't get the external to work, it would either crash with a rollup build error or just not work. I couldn't figure it out 😦

If you have a solution, I'll gladly take it!

Reply:
Ah I see the issue too. Getting:

```
[vite]: Rollup failed to resolve import "@astrojs/netlify/polyfill" from "/Users/bjorn/Work/oss/astro/packages/integrations/netlify/dist/netlify-functions.js".
```

Which should technically work but I'm not sure why not (maybe it's a monorepo issue). Maybe Matthew's idea of injecting the import in `generateBundle` or `renderChunk` for the output entry would be the way.